backend/tournesol/views/polls.py

```diff
@@ -12,7 +25,100 @@ class PollsView(RetrieveAPIView):
     """
     Retrieve a poll and its related criteria.
     """
+
     permission_classes = []
     queryset = Poll.objects.prefetch_related("criteriarank_set__criteria")
     lookup_field = "name"
     serializer_class = PollSerializer
+
+
+@extend_schema_view(
+    get=extend_schema(
+        description="Retrieve a list of recommended videos, sorted by decreasing total score.",
+        parameters=[
+            RecommendationsFilterSerializer,
+            OpenApiParameter(
+                "weights",
+                OpenApiTypes.OBJECT,
+                style="deepObject",
+                description="Weights for criterias in this poll."
+                " The default weight is 10 for each criteria.",
+                examples=[
+                    OpenApiExample(
+                        name="weights example",
+                        value={
+                            "reliability": 10,
+                            "importance": 10,
+                            "ignored_criteria": 0,
+                        }
+                    )
+                ]
+            ),
+        ],
+    )
+)
+class PollsRecommendationsView(ListAPIView):
+    permission_classes = []
+    pagination_class = LimitOffsetPagination
```

Comment:
I think the `pagination_class` can be removed here, as the current value `LimitOffsetPagination` is our default value configured in the settings. The `ListAPIView` being a child of `GenericAPIView`, `PollsRecommendationsView` should automatically inherit of this default value.

See:
https://github.com/encode/django-rest-framework/blob/efc7c1d664e5909f5f1f4d07a7bb70daef1c396e/rest_framework/generics.py#L46

```suggestion
```